pkg/image/view.go

```diff
@@ -126,13 +133,11 @@ func NewView(area *sdl.Rect, fileName string, bbComms chan<- comms.Image, toolCo
 
 // Destroy frees all assets acquired by the ui.Component
 func (iv *View) Destroy() {
-	gl.DeleteTextures(1, &iv.textureID)
-	gl.DeleteBuffers(1, &iv.vboID)
-	gl.DeleteVertexArrays(1, &iv.vaoID)
-	gl.DeleteBuffers(1, &iv.selVbo)
-	gl.DeleteVertexArrays(1, &iv.selVao)
-	gl.DeleteProgram(iv.programID)
-	gl.DeleteProgram(iv.selProgramID)
+	iv.texture.Delete()
+	iv.imgBuf.Destroy()
+	iv.selBuf.Destroy()
+	iv.program.Delete()
+	iv.selProgram.Delete()
```

Comment:
These should all use the same diction. Destroy would match what we've been doing in our interfaces, but Delete more closely matches OpenGL terminology.

Reply:
done